org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/ChangeConverter2.java

```diff
@@ -163,17 +130,7 @@ protected void _handleReplacements(ITextDocumentChange change) {
 	}
 
 	protected Object addTextEdit(String theUri, Document document, TextEdit... textEdits) {
```

Comment:
Please refactor to void method, too. Probably fallout from some Xtend to Java rewrite in the past. line 124 can always return null.

Reply:
i assume the problem is that the doRead calls want a return value. what do you propose?

Reply:
I'd let this particular doRead either return null or make it an IUnitOfWork.Void

Reply:
i guess an explict return null would do the latter

Reply:
this would be a breaking change if someone relies on it, but if it is fine for you, it is for me as well, so I have done the change. Thanks